### PR TITLE
Add sqare(d)->square(d)

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -26370,6 +26370,8 @@ spsaces->spaces
 sptintf->sprintf
 spurios->spurious
 spurrious->spurious
+sqare->square
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:26373]: Add -s variant

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -26370,6 +26370,8 @@ spsaces->spaces
...
+sqared->squared
 sqaure->square
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:26375]: You could add -s and -d off this typo too

**Review by peternewman — CHANGES_REQUESTED** [review]
A few comments

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:26373]: aa43e24

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:26376]: 80337f9

**Review by peternewman — APPROVED** [review]
LGTM

**Pull request merged**